src/modelio.cs

```diff
@@ -340,6 +345,11 @@ interface MDLLight
 		[Export ("lightType")]
 		MDLLightType LightType { get; set; }
 
+		// Added in iOS 10 SDK but it is supposed to be present in iOS 9.
+		[Mac (10,12)]
+		[Export ("colorSpace")]
+		string ColorSpace { get; set; }
```

Comment:
that's a weird choice for a type, i.e. it's not a `CGColorSpace`. Can you double check the online documentation and the header file to see if there's any detail on its usage ?

I suspect it might be constants, and `NSString` might be better in such case to avoid conversions, but I don't recall seeing field-based colourspace constants...